activemodel/test/models/aircraft.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Aircraft
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :manufactured_at, :datetime, default: -> { Time.current }
+  attribute :name, :string
+  attribute :wheels_count, :integer, default: 0
+  attribute :wheels_owned_at, :datetime
+end
```

Comment:
Since this model is only used in `activemodel/test/cases/normalized_attribute_test.rb`, I think we can just define it there, alongside `NormalizedAircraft`.